f8a_notification/entryscript.py

```diff
@@ -282,18 +288,15 @@ def generate_notification_payload():
             tmp_json['data']['attributes']['custom']['version_updates'] \
                 = repo_data['version_updates']
             final_payload.append(tmp_json)
-    print("<---Repo Data--->")
-    print(REPO_DATA)
-    print("<---Package Data--->")
-    print(PACKAGE_DATA)
-    print("<---New Version Data--->")
-    print(NEW_VERSION_DATA)
-    print("<---Version Data--->")
-    print(VERSION_DATA)
-    print("<---Final Data--->")
```

Comment:
Dont remove these statements. If not print, change them to logger but we need these to at least understand whats happening in the cron job